_drafts/SC2-1-2-no-keyboard-trap-standard-navigation.md

```diff
@@ -0,0 +1,106 @@
+---
+name: No keyboard trap standard navigation
+group:
+- SC2-1-2-no-keyboard-trap-standard-navigation
+- SC2-1-2-no-keyboard-trap-non-standard-navigation
+
+description: |
+  This rule checks if it is possible to use standard keyboard navigation to navigate through all content on a web page without becoming trapped in any element.
+
+success_criterion: 
+- 2.1.2 # No Keyboard Trap
+
+test aspects:
+- DOM Tree
+
+authors:
+- Dagfinn Rømen
+- Geir Sindre Fossøy
+- Malin Øvrebø
+- Shadi Abou-Zahra
+- Carlos Duarte
+- Anne Thyme Nørregaard
+- Stein Erik Skotkjerra
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element on a web page that is [focusable][] by keyboard.
+
+### Expectation
+
+For each target element focus can cycle to the browser UI by using [standard keyboard navigation][].
+
+**Note**: Cycling back to the browser UI can be done both by moving forward through the tab order and by moving backwards. It is not possible to fulfil this expectation by using browser specific shortcuts to return to the browser UI.
+
+## Assumptions
+
+- The WCAG success criterion applies to all content where focus can be moved to through keyboard navigation.
+- It is possible to use unmodified arrow or tab keys, or other standard exit methods to move focus away.
+- The focus order in keyboard navigation is cyclical, not linear, meaning that the focus order will cycle to the first/last element when it moves away from the last/first element.
```

Comment:
There's another assumption here that the Browser UI is part of the focus navigation cycle of the page. If it isn't and you need a keyboard shortcut to get out of the page and into the browser UI, the rule doesn't work. We need to test how this works with mobile browsers.

Reply:
We have tested a webpage with Chrome on Android. The browser UI is part of the focus navigation cycle. We will add this assumption.